docs/user/data_management.rst

```diff
@@ -186,6 +186,30 @@ user data using the hint ``geomesa.z3.interval``:
 
 See below for alternate ways to set the user data.
 
+.. _customizing_xz_index:
+
+Customizing the XZ-Index
```

Comment:
yeah, it's explained at the top of this page: http://www.geomesa.org/documentation/user/data_management.html#index-structure